deployment/conf/supported_apps_w_extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "apps": {
-    "sra_reads": {
+    "SRA Reads": {
+      "id": "sra_reads",
```

Comment:
I don't really understand the purpose of the ID change other than it's more readable - is there anything else?

Reply:
Yea I should have made it clearer. It turns out that there is a specific mapping that exists already in the narrative https://github.com/kbase/narrative/kbase-extension/static/kbase/config/staging_upload.json